### ISSUE TITLE
Add name of thread prefix into a title on mouse hover

### ISSUE THREAD

**Issue by Eldenroot** [dragonexpert/recentthreads#84]

**Comment by dragonexpert** [comment]
I'm not sure about this idea.  it would require javascript to do, but I also know that sometimes elements are finicky with what area you are hovering over.

**Comment by Eldenroot** [comment]
@dragonexpert - I dont think so, you can get the prefix name directly from MyBB... I use it in one of my modified plugin - check this: https://github.com/mybbgroup/Filterator/blob/master/inc/plugins/PrefixFilter.php

**Comment by dragonexpert** [comment]
Right now there is a setting for showing thread prefixes so I'm not sure on what the purpose of this is.

**Comment by Eldenroot** [comment]
Yes, we have... but imagine situation when you use only icons for prefixes... so the name of prefix would be displayed on mouse hover...
